pinc/Round.inc

```diff
@@ -620,3 +623,13 @@ function get_round_param(array $arr, string $key, ?Round $default = null, bool $
         }
     }
 }
+
+function get_OCR_pseudoround()
+{
+    $round = new stdClass();
+    $round->round_number = 0;
+    $round->id = "OCR";
+    $round->text_column_name = 'master_text';
+    $round->user_column_name = "'none'";  // string literal, not column name
```

Comment:
I don't know if this applies here, but when we skip rounds or add blanks, the user column is assigned the value `[none]` including the square brackets (in case someone comes along and registers that username?). AFAIK, we don't have a list of reserved usernames, do we? Just acceptable characters.

Reply:
This value was taken from the existing code this replaced, although I'm pretty sure that this string is actually never shown in the Page Details page. My inclination is to keep this as-is.

Reply:
TIL about `stdClass`.

TBH, I'm a little bit worried that playing dynamic tricks like this is going to cause bugs further down the line where we extend/refactor `Round` and forget to update `get_OCR_pseudoround`. It also makes the return type of `get_round_param()` into a white lie.

Looking at the implementation of `Round` and `Stage`, I'm not sure what the best way to handle this is.

Maybe just the simplest thing is to have a unit test that checks that all the attributes of OCR are also attributes of a real Round object.

Reply:
At a minimum could we add a comment to the `@return` of `get_round_param()` to warn about OCR having limited attributes?

Reply:
Yeah, it doesn't feel quite right to munge `get_round_param()` like this. We could instead create `get_round_or_ocr_param()` which basically just wraps the former and handles `OCR`. That would obviate the worry that this would pollute other users and ensure the burden of dealing with the OCR snowflake is on the script using it.